templates/create-daml-app-test-resources/index.test.ts

```diff
@@ -266,7 +268,9 @@ test('log in as three different users and start following each other', async ()
   await follow(page2, party3);
 
   // Check the following list is updated correctly.
-  await page2.waitForSelector('.test-select-following');
+  await page2.waitForFunction(
+    () => document.querySelectorAll(".test-select-following").length == 2
+  );
```

Comment:
> waitForSelector returns immediately if the selector is already present (which is documented). ...

`page2` is a separate result of `newUiPage` than `page1`. Do I understand correctly that the presence of `.test-select-following` on `page1` influences the query on `page2`? How is that happening?

Reply:
I’m not following, we waited on `page2` before, we wait on `page2` now.

Reply:
Hmm, nevermind, I misread the description of the issue.